packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app/sign-in-page/react-native/signin.md

```diff
@@ -0,0 +1,32 @@
+The async `signIn` method will automatically redirect users to your Okta organization for authentication. It will emit an event once a user successfully signs in. Make sure your event listeners are mounted and unmounted.
+**Note:** On iOS there isn't an `onCancelled` event. If the sign in process is cancelled, `onError` will be triggered.
```

Comment:
hypenated sign-in, as it is describing process here and removed "will be" and replaced with "is"

**Note:** On iOS there isn't an `onCancelled` event. If the sign-in process is cancelled, `onError` is triggered.